src/GeneralStateTestsFiller/stEWASMTests/createRevertFiller.yml

```diff
@@ -0,0 +1,142 @@
+# create failing with revert
```

Comment:
Is this now a duplicate of #68 ?

Reply:
Yes it's duplicated, I will remove the one in this PR if you agree